kms/spec/kms_spec.rb

```diff
@@ -163,6 +214,31 @@ def add_test_member_to_keyring_policy project_id:, key_ring_id:, member:, role:,
     kms_client.set_key_ring_iam_policy resource, policy_request
   end
 
+  def remove_test_member_to_keyring_policy project_id:, key_ring_id:, member:, role:, location:
+    kms_client = create_service_client
+
+    policy = get_test_keyring_policy(
+      project_id: project_id,
+      key_ring_id: key_ring_id,
+      location: location
+    )
+
+    resource = "projects/#{project_id}/locations/#{location}/" +
+               "keyRings/#{key_ring_id}"
+
+    if policy.bindings
+      policy.bindings.delete_if do |binding|
+        binding.role.include? role and binding.members.include? member
```

Comment:
As a general rule, don't use the `and` or `or` keywords.  They are not aliases for `&&` and `||` and they can often cause unexpected hair pulling.

They are banned in the GitHub styleguide.  Search for "keywords are banned" : [GitHub Ruby Styleguide](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md)

See also:
 - http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/
 - https://github.com/bbatsov/ruby-style-guide/issues/194

Use `&&` and `||` instead.  I know... they're not as pretty... darn you Ruby!